src/app/ApplicationTemplate.Shared/Presentation/Diagnostics/DiagnosticsOverlayViewModel.cs

```diff
@@ -22,6 +25,18 @@ public partial class DiagnosticsOverlayViewModel : ViewModel
 			await this.GetService<ISectionsNavigator>().OpenModal(ct, () => new DiagnosticsPageViewModel());
 		});
 
+		public IDynamicCommand NavigateToHttpTracingDiagnosticsPage => this.GetCommandFromTask(async ct =>
+		{
+			if (this.GetService<ISectionsNavigator>().GetActiveViewModel() is HttpTracingDiagnosticsPageViewModel vm)
+			{
+				await vm.NavigateBack.Execute(ct);
+			}
+			else
+			{
+				await this.GetService<ISectionsNavigator>().OpenModal(ct, () => new HttpTracingDiagnosticsPageViewModel());
```

Comment:
Instead of being in a page, I think it would make more sense to have that view within the diagnostics overlay. The navigation system can have an impact on ViewModel behavior.
Ex: A PageViewModel could unsubscribe from some observables when it's not the active one. By opening this as a modal, it changes the active page.
Diagnostics should not impact the app's behavior (because their goal is to observe said app behavior :P).